### PR TITLE
feat: finalise QR error messages

### PR DESCRIPTION
**Pull request by dalsontws**
Finalised QR error message
image

### CODE REVIEW

File: src/components/Login/utils.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ export const decodeQr = (code: string): DecodedQrResponse => {
   } catch (e) {
     if (e.message.includes("Unexpected token"))
       throw new Error(ERROR_MESSAGE.AUTH_FAILURE_EXPIRED_TOKEN);
+    // if user is not an operator
+    if (e.message.includes("Unauthorized auth token"))
+      throw new Error(ERROR_MESSAGE.AUTH_FAILURE_INVALID_TOKEN);
```
Comment (ddwsd) [inline src/components/Login/utils.ts:27]: hmm in the decode QR stage, it is not possible to tell the role of the user, so i don't think this block should be here

**Review by ddwsd — COMMENTED** [review]
i feel like this issue can be handled in #148 instead (the error message changes can be kept here tho) 😅

**Review by ddwsd — APPROVED** [review]
LGTM!

File: src/components/Login/utils.ts (hunk 1 above)
Comment (dalsontws) [reply src/components/Login/utils.ts]: got it, removed!

**Pull request merged**